ddp-workspace/projects/ddp-dsm-ui/src/app/FON/fon.module.ts

```diff
@@ -82,31 +76,5 @@ const layoutComponents = [NavigationComponent];
 })
 
 export class fonModule {
-  constructor(private storeService: StoreService,
-              private translateService: TranslateService,
-              private sessionService: SessionService,
-              private dssSessionService: SessionMementoService,
-              private title: Title,
-              private jwtHelper: JwtHelperService
-              ) {
-
-    const LOCALE = 'en';
-
-    // Title
-    this.title.setTitle('Fon');
-
-    // Token
-    const DSMToken = this.sessionService.getDSMToken();
-    const userGuid = this.jwtHelper.decodeToken(DSMToken)['https://datadonationplatform.org/cid'];
-    this.sessionService.setExpirationTime(DSMToken);
-    this.dssSessionService.setSession(null, DSMToken, userGuid, LOCALE, +this.sessionService.getTokenExpiration());
-
-    // Store
-    this.storeService.setStudy = MainConstants.study;
-    this.storeService.dispatchGetSettings(MainConstants.participantsListParent);
-
-    // Translation
-    this.translateService.setDefaultLang(LOCALE);
-    this.translateService.use(LOCALE);
-  }
+  constructor() {}
```

Comment:
We don't  need it more, right?

Reply:
Not here, but in fon.component, as that is initializing always when you log in to FON.